library/src/com/google/maps/android/data/kml/KmlLayer.java

```diff
@@ -50,7 +50,9 @@ public KmlLayer(GoogleMap map, InputStream stream, Context context)
         stream.close();
         mRenderer.storeKmlData(parser.getStyles(), parser.getStyleMaps(), parser.getPlacemarks(),
                 parser.getContainers(), parser.getGroundOverlays());
+        storeRenderer(mRenderer);
```

Comment:
Same comment as `geoJsonRenderer` - I believe `mRenderer` should be converted to a local variable (instead of member variable) here?  `KmlLayer` extends `Layer`, and there is a reference to the `KmlRenderer` superclass `Renderer` within `Layers.storeRenderer()`.